src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/controllers/UserController.kt

```diff
@@ -54,4 +58,35 @@ class UserController(private val userService: UserService) {
     userService.delete(id)
     return ResponseEntity(HttpStatus.NO_CONTENT)
   }
+
+  @CrossOrigin(origins = ["http://localhost:3000"])
+  @GetMapping("/{userId}/libraries")
+  fun getAllUserLibraries(
+    @PathVariable("userId") userId: String,
+  ): ResponseEntity<List<LibraryResponse>> {
+    val user = userService.getSingle(userId) ?: return ResponseEntity(HttpStatus.NOT_FOUND)
+    return ResponseEntity.ok(userService.convertUserToResponse(user).libraries)
+  }
+
+  @CrossOrigin(origins = ["http://localhost:3000"])
+  @PostMapping("/{userId}/libraries")
+  fun createUserLibrary(
+    @PathVariable("userId") userId: String,
+    @RequestBody libraryRequest: LibraryRequest
+  ): ResponseEntity<UserResponse> {
+    val userResponse = userService.createUserLibrary(userId, libraryRequest)
+      ?: return ResponseEntity(HttpStatus.NOT_FOUND)
+    return ResponseEntity.ok(userResponse)
+  }
+
+  @CrossOrigin(origins = ["http://localhost:3000"])
+  @PatchMapping("/{userId}/libraries/{libraryId}")
+  fun deleteUserLibrary(
+    @PathVariable("userId") userId: String,
+    @PathVariable("libraryId") libraryId: String
+  ): ResponseEntity<UserResponse> {
+    val userRespsonse = userService.deleteUserLibrary(userId, libraryId)
```

Comment:
Super nit and probably just worth addressing as a driveby somewhere else but :')
```suggestion
    val userResponse = userService.deleteUserLibrary(userId, libraryId)
```